ga4gh/exceptions.py

```diff
@@ -180,6 +187,60 @@ def getMessage(self):
             self.callSetId, self.variantSetId)
         return message
 
+
+class DataException(BaseServerException):
+    message = "Faulty data found."
+
+
+class EmptyDirException(DataException):
+    message = "Diretory empty, no VCF/BCF file was found"
+
+
+class MalFormedException(DataException):
+    message = "Faulty entry found in data file."
+
+
+class NotIndexedException(MalFormedException):
+    def __init__(self, fileName):
+        self.fileName = fileName
+
+    def getMessage(self):
+        self.message = "File {} must be indexed.".format(self.fileName)
+        return self.message
+
+
+class OverlappingException(MalFormedException):
```

Comment:
This name doesn't really tell me much when I don't know the context. What about `OverlappingVcfException`? For a user, this might be very mysterious also. What about:
```python
class OverlappingVcfException(MalformedInputException):
    """
    Exception thrown when two VCF files within a VariantSet directory contain records
    for the same contig. 
    """
    def __init__(self, fileName, contig):
         self.message = (
               "VCF file '{}' contains records for contig '{}'. Other files in this "
               "VariantSet have records for this contig, and overlapping VCFs "
              "are not permitted.")
```